CHANGELOG.md

```diff
@@ -7,6 +7,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.2.0] - 2025-01-15
+
+### Added
+
+* `left` and `right` fields support in `IosSafeArea`
```

Comment:
best practice is one entry per PR (this also emphasizes splitting into logical PRs)